### ISSUE TITLE
Project missing sincere show of thanks

### ISSUE THREAD

**Issue by FoxxMD** [Maldela/fancontrol-gui#48]
It's hard to be an open-source maintainer when the only activity you see on your project is people having issues/negative comments.

Let's use this issue as a place to show our thanks for this awesome tool! :heart: 

---

Instructions were clear and installation/running worked perfectly on Ubuntu 20.04. The little details and polish in this app are not overlooked; it shows up in the application without extra setup, inclusion of profiles, a helpful graph drawer with real-time data, and inclusion of gui buttons to control running as a service are all excellent and I am extremely grateful for them.

Thanks for making such an awesome tool, for free, for the community.

**Comment by JojiiOfficial** [comment]
I agree. I really like it even though I don't need it that much since I configure the sys fans inside the bios but for configuring the GPU fans it works perfectly too!
I really appreciated the `DNO_SYSTEMD` option since I don't use it as init and the DBus error was annoying.

I like really like this project and was impressed of how good it works. Thank you!

**Comment by Maldela** [comment]
Thanks guys. I really appreciate it. 
I know I haven't given enough love to this project over the last months. But I also have good news.
I'm working on a replacement for the fancontrol script with a bit more functionality and without some of the problems.
If you have any suggestions or feature requests, please let me know.

**Comment by JoneKone** [comment]
Oh :) I love this tool thx for making it..

**Comment by subterfugium** [comment]
> Thanks guys. I really appreciate it.
> I know I haven't given enough love to this project over the last months. But I also have good news.
> I'm working on a replacement for the fancontrol script with a bit more functionality and without some of the problems.
> If you have any suggestions or feature requests, please let me know.

I would like to control my case fans based on CPU or GPU temperature. If CPU or GPU temp goes over certain threshold, certain fan% is applied. If either card is under heavy load, increase case fans to circulate more air.

Basically, add function for curve to read two input temperatures OR
create virtual sensors, for example, max(CPU_temp, GPU_temp) or min(CPU_temp, GPU_temp), etc...

**Comment by Maldela** [comment]
> > Thanks guys. I really appreciate it.
> > I know I haven't given enough love to this project over the last months. But I also have good news.
> > I'm working on a replacement for the fancontrol script with a bit more functionality and without some of the problems.
> > If you have any suggestions or feature requests, please let me know.
> 
> I would like to control my case fans based on CPU or GPU temperature. If CPU or GPU temp goes over certain threshold, certain fan% is applied. If either card is under heavy load, increase case fans to circulate more air.
> 
> Basically, add function for curve to read two input temperatures OR
> create virtual sensors, for example, max(CPU_temp, GPU_temp) or min(CPU_temp, GPU_temp), etc...

Great idea!

**Comment by JoneKone** [comment]
dunno if it's possible, but i would want to rename fans. If not possible in kernel level, then even in gui(program) level. Also if I'm not using a fangui for certain fan, i could be hidden in the "Fans" list, a setting that lets you choose witch fans are showing in the list?

**Comment by Maldela** [comment]
> dunno if it's possible, but i would want to rename fans. If not possible in kernel level, then even in gui(program) level. Also if I'm not using a fangui for certain fan, i could be hidden in the "Fans" list, a setting that lets you choose witch fans are showing in the list?

You can already rename fans by clicking on their name above the curve graph. The cursor changes when you hover over the right place.

**Comment by JoneKone** [comment]
> > dunno if it's possible, but i would want to rename fans. If not possible in kernel level, then even in gui(program) level. Also if I'm not using a fangui for certain fan, i could be hidden in the "Fans" list, a setting that lets you choose witch fans are showing in the list?
> 
> You can already rename fans by clicking on their name above the curve graph. The cursor changes when you hover over the right place.

Oh never realized that was there.

**Comment by subterfugium** [comment]
> > dunno if it's possible, but i would want to rename fans. If not possible in kernel level, then even in gui(program) level. Also if I'm not using a fangui for certain fan, i could be hidden in the "Fans" list, a setting that lets you choose witch fans are showing in the list?
> 
> You can already rename fans by clicking on their name above the curve graph. The cursor changes when you hover over the right place.

This is also news to me :) Good feature. However, I would like to see fan names to be read from lm-sensors. What I do is I create custom config file to [rename sensors](https://wiki.archlinux.org/index.php/lm_sensors#Example_2._Renaming_labels) (for example, /etc/sensors.d/renames):
```
chip "it8728-isa-*"
  label fan1 "CPU Rad fans"
  label fan2 "SYS1 Top fan"
  label fan3 "SYS2 no fan"
  label fan4 "SYS3 RGB fans (no rpm)"
  label fan5 "CPU Pump"
```
These fan names are then available via _sensors_:
```
it8728-isa-0a40
CPU Rad fans:            136 RPM  (min =    0 RPM)
SYS1 Top fan:            210 RPM  (min =    0 RPM)
SYS2 no fan:               0 RPM  (min =    0 RPM)
SYS3 RGB fans (no rpm):    0 RPM  (min =    0 RPM)
CPU Pump:               1979 RPM  (min =    0 RPM)
```
and PSensor:
![image](https://user-images.githubusercontent.com/4238921/98718745-83035680-2397-11eb-87f5-3113b6e24000.png)

Ideally would be nice to see same names in fancontrol gui as well. However, it's good that fancontrol gui already has renaming functionality.

**Comment by mediocre-to** [comment]
Useful tool! Thanks a lot! Will be interesting to know about developments!  Thanks so far!

(Ubuntu 20.04 on Intel/asus)

**Comment by epifanio** [comment]
Hey! thank you for this - I am finding it extremely useful.
